service/src/test/java/bio/terra/workspace/service/resource/controlled/cloud/azure/disk/CreateAzureDiskStepTest.java

```diff
@@ -32,7 +32,7 @@
 import org.mockito.ArgumentCaptor;
 import org.mockito.Mock;
 
-public class CreateAzureDiskStepTest extends BaseAzureConnectedTest {
+public class CreateAzureDiskStepTest extends BaseAzureUnitTest {
```

Comment:
Made a bunch of Azure step tests unit instead of connected, I think that was the intent and it is just a misconfiguration.